docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -57,16 +65,38 @@ SELECT TRANSFORM ( expression [ , ... ] )
 
     Specifies a command or a path to script to process data.
 
-### SerDe behavior
+### ROW FORMAT DELIMITED BEHAVIOR
+
+When spark use `ROW FORMAT DELIMITED` format, Spark will use `\u0001` as default filed delimit,
+use `\n` as default line delimit and use `"\N"` as `NULL` value in order to differentiate `NULL` values 
```

Comment:
This is too long to read. Can we list up in bulletpoints?

Reply:
yea, +1 for the @HyukjinKwon suggestion. The current sentence is hard to read...

Reply:
How about current